index.js

```diff
@@ -18,41 +22,36 @@ var suffix = new RegExp(qs)
 module.exports = Nanorouter
 
 function Nanorouter (opts) {
+  if (!(this instanceof Nanorouter)) return new Nanorouter(opts)
   opts = opts || {}
+  this.router = wayfarer(opts.default || '/404')
+}
+
+Nanorouter.prototype.on = function onRoute (routename, listener) {
```

Comment:
The `onRoute` variable name doesn't do anything in this code. Could we perhaps remove the variable names from all methods, and only keep the method names?

Reply:
I've just added it to have a named function.
But usually nobody does this for prototype.
ES6 improved anonymous functions a lot by defining the name automatically for the new syntaxes and `let` and `const`. But as we're still using prototypes, I'm good with removing it.
This module is quite small anyways and the function name of the constructor might be good enough.